packages/manager/apps/vrack-services/src/pages/create/VrackConfirmModal.tsx

```diff
@@ -63,6 +63,11 @@ export const VrackConfirmModal: React.FC<Props> = ({
         variant={ODS_BUTTON_VARIANT.ghost}
         color={ODS_THEME_COLOR_INTENT.primary}
         onClick={onCancel}
+        onKeyDown={(event: React.KeyboardEvent) => {
+          if ([' ', 'Enter'].includes(event.key)) {
```

Comment:
Maybe you can put this array inside an const (or utils file) and reuse it
````
const KEYBOARD_ACTION = [' ', 'Enter']
...
if (KEYBOARD_ACTION.includes(event.key)) {
...
```

Reply:
https://github.com/ovh/manager/pull/10418